Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/StartRide.hs

```diff
@@ -150,11 +152,22 @@ startRide ServiceHandle {..} rideId req = withLogTag ("rideId-" <> rideId.getId)
       case dashboardReq.point of
         Just point -> pure point
         Nothing -> do
-          driverLocation <- findLocationByDriverId driverId >>= fromMaybeM LocationNotFound
+          enableLocationTrackingService <- asks (.enableLocationTrackingService)
+          driverLocation <- do
+            if enableLocationTrackingService
+              then do
+                ltsCfg <- asks (.ltsCfg)
+                driverLocations <- LF.driversLocation ltsCfg [driverId]
+                LF.findByDriverId driverLocations driverId & fromMaybeM LocationNotFound
+              else findLocationByDriverId driverId >>= fromMaybeM LocationNotFound
           pure $ getCoordinates driverLocation
-
+  enableLocationTrackingService <- asks (.enableLocationTrackingService)
```

Comment:
This config can be fetched only once? same lines of code is repeated in the same function

Reply:
removed